solidity-its/src/main/java/org/sonarsource/solidity/its/SolidityRulingIts.java

```diff
@@ -34,6 +34,16 @@ public void addIssue(Token start, Token stop, String reportMessage, String exter
     reportIssue(start.getLine(), projectDir);
   }
 
+  @Override
+  public void addIssueOnFile(String reportMessage, String externalRuleKey) {
+    // TODO
+  }
+
+  @Override
+  public void addIssue(Token start, Token stop, int offset, String reportMessage, String externalRuleKey) {
+    // TODO Auto-generated method stub
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=33&issues=AWRuvWkpI66nRh8YCYJn&open=AWRuvWkpI66nRh8YCYJn)

[ISSUE_KEY]: AWRuvWkpI66nRh8YCYJn